apps/condo/domains/common/components/MultipleFileUpload.tsx

```diff
@@ -164,32 +169,30 @@ const MultipleFileUpload: React.FC<IMultipleFileUploadProps> = (props) => {
                         if (!id) {
                             // remove file that failed to upload from list
                             setListFiles([...listFiles].filter(file => file.uid !== uid))
-                            return 
+                            return
                         }
                         setListFiles([...listFiles].filter(file => file.id !== id))
                         onFilesChange({ type: 'delete', payload: file })
                     }} />
                 )
                 return removeIcon
             },
-        }, 
+        },
         customRequest: (options: UploadRequestOption) => {
             const { onSuccess, onError } = options
             const file = options.file as UploadFile
             if (file.size > MAX_UPLOAD_FILE_SIZE) {
                 const error = new Error(FileTooBigErrorMessage)
                 onError(error)
-                setListFiles([...listFiles.filter(file => file.status !== 'uploading'), { name: file.name, uid: file.uid, id: null, status: 'error', error: error }])
-                return 
+                return
             }
             return createAction({ ...initialCreateValues, file }).then( dbFile  => {
                 const [uploadFile] = convertFilesToUploadFormat([dbFile])
-                setListFiles([...listFiles.filter(file => file.status !== 'uploading'), uploadFile])
-                onSuccess('Ok', null)
-                onFilesChange({ type: 'add', payload: dbFile })                                
+                onSuccess(uploadFile, null)
+                onFilesChange({ type: 'add', payload: dbFile })
             }).catch(err => {
                 const error = new Error(UploadFailedErrorMessage)
-                setListFiles([...listFiles.filter(file => file.status !== 'uploading'), { name: file.name, uid: file.uid, id: null, status: 'error', error: error }])
+                console.error('Upload failed', err)
```

Comment:
I don't know about the policy of console.log messages in our project. So I don't know if this is ok for us to leave console.logs in code

Can you please point me to the source where I can read about it? 🥺

Reply:
Some times we are facing different errors that are not supposed to happen. For example, when we had Firebase we were catching all (4) documented errors. But we cought the  undocumented one. And we can't reproduce it repeating user's actions and we can't ask the user to give us error from console. So, I suppose when we are dealing with third party libraries it's good to show strange errors at least in console